plugin/storage/integration/kafka_test.go

```diff
@@ -40,6 +42,9 @@ func (s *KafkaIntegrationTestSuite) initialize(t *testing.T) {
 	topic := "jaeger-kafka-integration-test-" + strconv.FormatInt(time.Now().UnixNano(), 10)
 
 	f := kafka.NewFactory()
+	t.Cleanup(func() {
+		assert.NoError(t, f.Close())
+	})
```

Comment:
Little concerned about this, whether cleanup should be called here or after initializing it?

Reply:
usually factories in Jaeger don't do anything upon creation and only start opening connections after initializing. So yes, it should be done later.